src/pages/publication/PublicationForm.tsx

```diff
@@ -160,7 +162,17 @@ const PublicationForm: FC<PublicationFormProps> = ({
     setTabNumber(tabNumber + 1);
   };
 
-  const savePublication = async (values: Publication) => {};
+  const savePublication = async (values: BackendPublication) => {
+    const updatedPublication = await updatePublication({
+      ...publicationValues,
+      entityDescription: values.entityDescription,
+    });
+    if (updatedPublication.error) {
+      dispatch(setNotification(updatedPublication.error, NotificationVariant.Error));
+    } else {
+      dispatch(setNotification(t('feedback:success.update_publication')));
```

Comment:
vi skulle kanskje satt responsen inn i formik igjen, så vi vet at verdiene er riktige. da må i put returnere hele objektet i stedet for bare id da, så vi kan ta det i en annen runde. kanskje lage en sak på det?

Reply:
har sagt ifra til jan om dette også, han skal returnere hele posten ved PUT 👍

Reply:
lagde en task på dette: NP-614